webui/react/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+import Icon from 'shared/components/Icon/Icon';
+import { dateTimeStringSorter } from 'shared/utils/sort';
+import { useAuth } from 'stores/auth';
+import { ShirtSize } from 'themes';
+import { CommandTask, DetailedUser, ExperimentItem, Project } from 'types';
+import { Loadable } from 'utils/loadable';
+
+const Dashboard: React.FC = () => {
+  const [experiments, setExperiments] = useState<ExperimentItem[]>([]);
+  const [tasks, setTasks] = useState<CommandTask[]>([]);
+  const [projects, setProjects] = useState<Project[]>([]);
+  const [submissions, setSubmissions] = useState<Submission[]>([]);
+  const [canceler] = useState(new AbortController());
+  const loadableAuth = useAuth();
+  const authUser = Loadable.match(loadableAuth.auth, {
+    Loaded: (auth) => auth.user,
+    NotLoaded: () => undefined,
+  });
+
+  type Submission = ExperimentItem & CommandTask;
+
+  const fetchTasks = useCallback(
+    async (user: DetailedUser) => {
+      const [commands, jupyterLabs, shells, tensorboards] = await Promise.all([
+        getCommands({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getJupyterLabs({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getShells({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getTensorBoards({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+      ]);
+      const newTasks = [...commands, ...jupyterLabs, ...shells, ...tensorboards];
+      setTasks(newTasks);
+    },
+    [canceler],
+  );
+
+  const fetchExperiments = useCallback(async (user: DetailedUser) => {
+    const response = await getExperiments({
+      orderBy: 'ORDER_BY_DESC',
+      sortBy: 'SORT_BY_START_TIME',
+      userIds: [user.id],
+    });
+    setExperiments(response.experiments);
+  }, []);
+
+  const fetchProjects = useCallback(async () => {
+    const projects = await getProjectsByUserActivity({ limit: 12 });
+    setProjects(projects);
+  }, []);
+
+  useEffect(() => {
+    fetchProjects();
+    if (!authUser) return;
+    fetchExperiments(authUser);
+    fetchTasks(authUser);
+  }, [authUser, fetchExperiments, fetchTasks, fetchProjects]);
+
+  useEffect(() => {
+    let submissions: Submission[] = [];
+    submissions = submissions
+      .concat(experiments as Submission[])
+      .concat(tasks as Submission[])
+      .sort((a, b) => dateTimeStringSorter(b.startTime, a.startTime));
+    setSubmissions(submissions);
+  }, [experiments, tasks]);
+
+  return (
+    <Page title="Home">
+      <Section title="Recent projects">
```

Comment:
i know its lowercase in figma, but it should be uppercase?
also, is it recently visited or recently modified projects?

Reply:
casing looks fine to me. it's recently visited.